### PR TITLE
OP-875 link bill admission

### PR DESCRIPTION
**Pull request by mwithi**
See OP-875.
Check fails because they need https://github.com/informatici/openhospital-core/pull/782.

### CODE REVIEW

File: rpt/BillsReportMonthlyWard.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Month = Month
+
+Receivable = Receivable
+
+Report.From = report from
+
+To = To
+
+Periodical.Incomes.Billed.And.Receivables.Report = Periodical Incomes/Billed and Receivables Report
```
Comment (dbmalkovsky) [inline rpt/BillsReportMonthlyWard.properties:15]:
```suggestion
Periodical.Incomes.Billed.And.Receivables.Report = Periodic Incomes/Billed and Receivables Report
```
The typical meaning of periodical is: a magazine or newspaper published at regular intervals.

File: rpt/BillsReportMonthlyWard_subreport.properties [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Month = Month
+
+Receivable = Receivable
+
+Report.From = report from
+
+To = To
+
+Periodical.Incomes.Billed.And.Receivables.Report = Periodical Incomes/Billed and Receivables Report
```
Comment (dbmalkovsky) [inline rpt/BillsReportMonthlyWard_subreport.properties:15]: same suggestion as above for `periodical`

**Review by dbmalkovsky — APPROVED** [review]
LGTM with small optional changes/suggestions

**Review by emecas — COMMENTED** [review]
Looks good to me, @mwithi  in a previous PR you mentioned to me that one of the checks  was not relevant, here the following is the current situation, how could we rationalize it?

image

File: rpt/BillsReportMonthlyWard.properties (hunk 1 above)
Comment (mwithi) [reply rpt/BillsReportMonthlyWard.properties:15]: Thanks!

Comment (mwithi) [reply rpt/BillsReportMonthlyWard.properties:15]: ok, but I need to change also the key, new commit comming!

**Comment by mwithi** [comment]
> Looks good to me, @mwithi in a previous PR you mentioned to me that one of the checks was not relevant, here the following is the current situation, how could we rationalize it?

Because in previous PR the core was already merged (my bad) so it was passing only `pull_request` check, but normally both fails in first sight. Once the core is merged `pull_request` update itself automatically, `push` check must be relaunched manually.

**Pull request merged**